src/libs/ReportUtils.ts

```diff
@@ -4255,7 +4255,7 @@ function canRequestMoney(report: OnyxEntry<Report>, policy: OnyxEntry<Policy>, o
     if (isMoneyRequestReport(report)) {
         const isOwnExpenseReport = isExpenseReport(report) && isOwnPolicyExpenseChat;
         if (isOwnExpenseReport && PolicyUtils.isPaidGroupPolicy(policy)) {
-            return isDraftExpenseReport(report);
+            return isDraftExpenseReport(report) || (PolicyUtils.isInstantSubmitEnabled(policy) && isProcessingReport(report));
```

Comment:
I believe at this point you can revert this change after pulling `main` - since this was just added in https://github.com/Expensify/App/pull/36997 - sorry!

Reply:
@Beamanator I'm thinking of removing that helper function, it seems unnecessary and the naming is verbose and a bit confusing. What do you think?

Reply:
What do you think?

Reply:
I see what you mean & yeah maybe the function name is kinda gross, but don't we reuse the same check (`PolicyUtils.isInstantSubmitEnabled(policy) && isProcessingReport(report)`) in many places & won't we continue to do so? So wouldn't it be nice to make a util function for this? I slightly prefer a util function IFF we can come up with a good name for it that's super clear 😅

Reply:
Sounds good, I can't think of a way to improve that function name, so I think I'll create a new one which will instead decide if we can add transactions to a report or not, similar to how we do it in Auth. Will commit the update and let me know what you think! I think it looks cleaner.